scss/mixins/_spinners.scss

```diff
@@ -0,0 +1,121 @@
+// Spinner variants
+//
+// Easily pump out default styles, sizes, styling variants
+
+@keyframes spinner {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@mixin spinner-size($size, $width: 0) {
+  $inside-size: ($size - 2 * $width);
+
+  min-width: $size;
+  min-height: $size;
+
+  &:before,
+  &.spinner-fallback .spinner-fallback-spin {
+    width: $inside-size;
+    height: $inside-size;
+    margin-top: (-$inside-size / 2);
+    margin-left: (-$inside-size / 2);
+    border-width: $width;
+  }
+}
+
+@mixin spinner-circle-variant($color-first, $color-second) {
+  &:not(.spinner-fallback):before,
+  &.spinner-fallback .spinner-fallback-spin {
+    border-style: solid;
+    border-color: $color-second;
+    border-top-color: $color-first;
+    border-radius: 50%;
+  }
+
+  &:not(:required):before {
+    animation: spinner $spinner-animation-duration linear infinite;
+  }
+}
+
+@mixin spinner-comet-variant($color) {
+  &:not(.spinner-fallback):before,
+  &.spinner-fallback .spinner-fallback-spin {
+    border-top-style: solid;
```

Comment:
Properties should be ordered border-top-color, border-top-style, border-right-color, border-right-style, border-radius